playbooks/robusta_playbooks/prometheus_enrichments.py

```diff
@@ -61,3 +64,32 @@ def prometheus_enricher(event: ExecutionBaseEvent, params: PrometheusQueryParams
     event.add_enrichment(
         [PrometheusBlock(data=prometheus_result, query=params.promql_query)],
     )
+
+
+def get_duplicate_kubelet_msg(rule_group) -> str:
+    """
+    checks if there are multiple kubelets and
+    """
+    NO_MESSAGE = ""
+    if not rule_group or "kubelet" not in rule_group:
+        return NO_MESSAGE
+    labels_selectors = "k8s-app=kubelet, app.kubernetes.io/managed-by=prometheus-operator"
+    v1 = client.CoreV1Api()
+    kubelet_services: List[V1Service] = v1.list_service_for_all_namespaces(label_selector=labels_selectors).items
```

Comment:
Is there a way to build a generic solution, that find this error for any duplicated service, rather than a hard coded solution only for kubelet?

Reply:
Duplicate services might not always cause this issue. (Specifically kubelet should always be scraped due to there being a ServiceMonitor for it)

<img width="261" alt="Screen Shot 2023-04-13 at 21 37 06" src="https://user-images.githubusercontent.com/97387909/231852419-4ac58418-e1a4-418a-857c-96dccac56cd2.png">

It would be better to use the prometheus targets api to verify the duplicate services are being scraped. I will update this code when I finish the target down enricher to make it more generic. (I already have code to get the prometheus targets there) I'll note to add it in that PR.